pkg/controller/perconaservermongodb/custom_users.go

```diff
@@ -119,3 +142,15 @@ func (r *ReconcilePerconaServerMongoDB) reconcileCustomUsers(ctx context.Context
 
 	return nil
 }
+
+func sysUserNames(sysUsersSecret corev1.Secret) map[string]struct{} {
+	sysUserNames := make(map[string]struct{}, len(sysUsersSecret.Data))
+
+	sysUserNames[string(sysUsersSecret.Data[api.EnvMongoDBClusterAdminUser])] = struct{}{}
+	sysUserNames[string(sysUsersSecret.Data[api.EnvMongoDBDatabaseAdminUser])] = struct{}{}
+	sysUserNames[string(sysUsersSecret.Data[api.EnvMongoDBUserAdminUser])] = struct{}{}
+	sysUserNames[string(sysUsersSecret.Data[api.EnvMongoDBBackupUser])] = struct{}{}
+	sysUserNames[string(sysUsersSecret.Data[api.EnvMongoDBClusterMonitorUser])] = struct{}{}
```

Comment:
wouldn't this be more readable if we do this with a for loop?

Reply:
Yes, but with a loop, you will get usernames and passwords and I just wanted usernames, but I guess it wouldn't be a problem if we hold passwords as well, security vise as well as perf vise since it is only a few items in there.